mantis-control-plane/mantis-control-plane-core/src/main/java/io/mantisrx/server/worker/TaskExecutorGateway.java

```diff
@@ -36,6 +37,8 @@ public interface TaskExecutorGateway extends RpcGateway {
      */
     CompletableFuture<Ack> submitTask(ExecuteStageRequest request);
 
+    CompletableFuture<Ack> cacheJobArtifacts(CacheJobArtifactsRequest request);
```

Comment:
nit: cache implies it can be cleanup/evicted but in this case, it's more persistent?

Reply:
maybe just add a document to this API for now?